srcCxx/ShapeType_main.cxx

```diff
@@ -255,7 +322,7 @@ int run(DomainId_t domain_id, bool use_security,
 
 
     if ( pub_topic != NULL ) {
-        writer = create_writer(participant, pub_topic, partition, livelinessPeriod);
+        writer = create_writer(participant, pub_topic, partition, livelinessPeriod, color);
```

Comment:
`create_writer` now accepts a color argument. Should `create_reader` also accept a last `color` input?

Reply:
I think that would be useful if we want to add tests that use a ContentFilter.
Otherwise, I don't see a reason (other than symmetry) for it now.

I can't recall if security impacts content-filtering... if it does, then it would make sense to add it.

Reply:
I was nitpicking for symmetry, I don't think it impacts any other way. Feel free to resolve the conversation.